### ISSUE TITLE
Can we have a hotkey that trigger all damage boosting potions?

### ISSUE THREAD

**Issue by AnanZero** [badplayerr/beta-autopot#29]
It would be really handy for pathfinder and other classes too. Please!

**Comment by lordskylare** [comment]
Yeah, would be useful for Diamond flask, Atziri flask, Lion' Roar etc

04.09.2016 20:28 "Sam Sung" <notifications@github.com> napisał(a):

**Comment by luminair** [comment]
Yes, I would also like Sulphur flask included too 👍
